### ISSUE TITLE
Feature Request: Electrum server

### ISSUE THREAD

**Issue by jaonoctus** [jamaljsr/polar#807]
## Description

Would be nice the option to have a docker instance of electrum server (electrs), connected to the bitcoin one.

I already have an docker-compose file to do that, but I think it is a time-saver if we have it on Polar itself. wdyt?

![image](https://github.com/jamaljsr/polar/assets/7918067/78b5d04d-42c5-4108-8a28-ed3c9afccf5e)

**Comment by jamaljsr** [comment]
Yes, I think it would be really cool to have additional bitcoin/LN related servers, such as electrs, added to Polar.

For some time now I've wanted to build out some sort of plugin/app platform that would make it easy to integrate additional services. Right now, it it pretty challenging to add other kinds of nodes because they are all hard-coded. I struggled with this head on when adding support for `tapd` nodes. It was a lot more work than I anticipated. Creating an app platform is a pretty big under-taking as well, which also comes with as additional maintenance burden of having to keep the new node types updated with their latests versions. 

Unfortunately, I don't have the spare time to tackle these updates myself but I have no objections to adding this if someone was interested in working on it.
